### PR TITLE
11.0 mig analytic tag dimension purchase warning

### PR DESCRIPTION
**Pull request by mpanarin**

### CODE REVIEW

File: analytic_tag_dimension_purchase_warning/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Analytic Tag Dimension Purchase Warning",
+    "summary": "Group Analytic Entries by Dimensions",
+    "category": "Generic Modules/Accounting",
+    "version": "11.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Camptocamp SA,Odoo Community Association (OCA)",
+    "website": "http://www.camptocamp.com",
```
Comment (simahawk) [inline analytic_tag_dimension_purchase_warning/__manifest__.py]: https://github.com/OCA/account-analytic

File: analytic_tag_dimension_purchase_warning/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Analytic Tag Dimension Purchase Warning",
+    "summary": "Group Analytic Entries by Dimensions",
+    "category": "Generic Modules/Accounting",
+    "version": "11.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Camptocamp SA,Odoo Community Association (OCA)",
```
Comment (simahawk) [inline analytic_tag_dimension_purchase_warning/__manifest__.py]: remove "SA"

**Review by simahawk — CHANGES_REQUESTED** [review]

File: analytic_tag_dimension_purchase_warning/__manifest__.py (hunk 2 above)
Comment (ghost) [reply analytic_tag_dimension_purchase_warning/__manifest__.py]: Separate authors w/ space please :smile:

File: analytic_tag_dimension_purchase_warning/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    "website": "http://www.camptocamp.com",
+    "depends": [
+        'analytic_tag_dimension',
+        'purchase'
```
Comment (ghost) [inline analytic_tag_dimension_purchase_warning/__manifest__.py]: Add a trailing comma

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by simahawk — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by simahawk** [comment]
NOTE: depends on https://github.com/OCA/account-analytic/pull/146 -> oca_dependencies.txt to be updated

File: oca_dependencies.txt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# list the OCA project dependencies, one per line
+# add a github url if you need a forked version
+account-analytic https://github.com/arkostyuk/account-analytic 11.0-mig-analytic_tag_dimension
```
Comment (grindtildeath) [inline oca_dependencies.txt]: As #146 is merged, you can remove this line.

**Review by grindtildeath — COMMENTED** [review]
oca_dependencies to update, otherwise LGTM ! 👍

**Review by yvaucher — APPROVED** [review]

**Pull request merged**